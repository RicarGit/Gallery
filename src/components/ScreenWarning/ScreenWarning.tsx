import * as S from './ScreenWarning.styled'

export type Warning = {
  emoji: string
  children: string
}

export const ScreenWarning = ({ emoji, children }: Warning) => {
  return (
    <S.ScreenWarning>
      <div className="emoji">{emoji}</div>
      {children}
    </S.ScreenWarning>
  )
}