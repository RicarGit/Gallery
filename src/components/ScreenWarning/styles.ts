import styled from "styled-components/macro"

export const ScreenWarning = styled.div`
  text-align: center;

  .emoji {
    font-size: 50px;
    margin-bottom: 20px;
  }
`