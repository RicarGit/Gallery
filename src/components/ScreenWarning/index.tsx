import * as S from './styles'
import { Warning } from '../../types/Warning'

export const ScreenWarning = ({ emoji, children }: Warning) => {
  return (
    <S.ScreenWarning>
      <div className="emoji">{emoji}</div>
      {children}
    </S.ScreenWarning>
  )
}