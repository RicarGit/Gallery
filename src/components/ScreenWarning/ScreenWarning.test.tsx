import { render } from '@testing-library/react'
import { ScreenWarning } from './ScreenWarning'

describe('PhotoCard', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <ScreenWarning emoji=''>
        {'children'}
      </ScreenWarning>
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})