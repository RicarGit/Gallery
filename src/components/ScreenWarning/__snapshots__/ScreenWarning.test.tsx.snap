// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoCard should match snapshot 1`] = `
<div
  class="ScreenWarningstyled__ScreenWarning-sc-1v9tbg9-0 zmBAE"
>
  <div
    class="emoji"
  />
  children
</div>
`;
